src/runtime/pkg/containerd-shim-v2/stream.go

```diff
@@ -125,7 +127,16 @@ func ioCopy(shimLog *logrus.Entry, exitch, stdinCloser chan struct{}, tty *ttyIO
 			shimLog.Debug("stdout io stream copy started")
 			p := bufPool.Get().(*[]byte)
 			defer bufPool.Put(p)
-			io.CopyBuffer(tty.io.Stdout(), stdoutPipe, *p)
+
+			for {
+				var _, err = io.CopyBuffer(tty.io.Stdout(), stdoutPipe, *p)
+				shimLog.Debug("stdout io stream copy error happens: error = %w", err.Error())
```

Comment:
@LindaYu17  I recommend verifying whether `err` is null before printing the error log. Otherwise there is a panic: https://github.com/kata-containers/kata-containers/actions/runs/8076580876/job/22066440187?pr=8798#:~:text=panic%3A%20runtime%20error,44

Reply:
fixed, thanks